README.md

```diff
@@ -1,14 +1,91 @@
 # Project Rheumaconnect
-### Helping Rheumatic patients easily access information related to their conditions within the ease of their homes
+Helping Rheumatic patients easily access information related to their conditions within the ease of their homes
 
-The main goal of this project is to help Rheumatic patients get relevant information related to their conditions without visiting the hospital clinic everytime they need something by building a tri-lingual information hub maintained by specialist doctors.
+---
 
-The development of the project is run by the Software Engineering Student Community of SLIIT, mainly with the support of fellow volunteers.
+## 🎯 Project Overview
+**RheumaConnect** is an innovative digital platform designed to bridge the information gap between rheumatic patients and medical professionals. By providing verified, accessible medical information in multiple languages, we aim to improve patient education and reduce unnecessary hospital visits for basic information needs.
 
-## If you are an enthusiast who would like to bring smiles to people's lives we invite you to take part in this effort by contributing ❤️
+### Core Services
+•Expert Backed Medical Information: Patients can access verified, up to date information about various rheumatic conditions, curated by specialist rheumatologists.
+•Smart Appointment System: patients easily be able to use booking system for consultations with rheumatology specialists.
+•Medication Guide: Patients can get comprehensive information about treatment options, medication details, and possible side effects.
+•Multilingual Support: All information available in three languages (English, Sinhala, Tamil) to ensure accessibility for all the patients.
 
-You can contribute to the project in 2 ways,
-- Technically (coding, ui-design)
-- Non-technically (documentation, graphic design, etc)
+### Why RheumaConnect?
+Rheumatic patients often face several challenges,
+
+• Limited access to reliable medical information about their conditions.
+• Difficulty in understanding complex medical terminology.
+• Long waiting times for specialist consultations.
+• Confusion about medication and treatment options.
+• Language barriers in accessing medical information.
+
+RheumaConnect is transforming rheumatic care by offering an integrated approach to patient support. The platform provides free access to expert verified medical resources, with easy to understand explanations of conditions and treatments. Patients can quickly book appointments, reducing waiting times, and access clear medication guidelines along with detailed treatment information. With multi-language support, all resources are available in patients' preferred languages, ensuring better understanding and promoting improved healthcare outcomes.
```

Comment:
Shall we remove "booking appointments" part since this mainly offers information only. But on later stages we can provide clinic open/close times, crowded status, etc...